kar/src/main/feature/feature.xml

```diff
@@ -69,7 +71,7 @@
         <bundle start-level="77" start="false">mvn:org.apache.unomi/cxs-geonames-rest/${project.version}</bundle>
         <bundle start-level="77" start="false">mvn:org.apache.unomi/cxs-privacy-extension-services/${project.version}</bundle>
         <bundle start-level="77" start="false">mvn:org.apache.unomi/cxs-privacy-extension-rest/${project.version}</bundle>
-        <bundle start-level="78" start="false">mvn:org.apache.unomi/unomi-rest/${project.version}</bundle>
+        <bundle start-level="78" start="true">mvn:org.apache.unomi/unomi-rest/${project.version}</bundle>
```

Comment:
Somehow unomi-rest can not start along with other bundle! therefore I force it over here!

Reply:
Are you using unomi:start command after Karaf startup ? If you are I thing you will not need to force the start in the feature file. 

The bundles are not started by default to allow for migrations to be possible.

Reply:
But it is possible to auto-start the bundles with a command line parameter however. 
-Dunomi.autoStart=true